cypress-io/cypress
# fix: Remove autocomplete from inputs in Cypress UI #22102

- Closes #22102 

### User facing changelog

Disables autocomplete for input fields.

### Additional details



### Steps to test

Open up Cypress App and type in the spec search filter. Click on a spec and navigate back, the autocomplete should not show in the spec search input.


### How has the user experience changed?

Before:
![171952206-e6c7536d-ba9d-4d5c-a518-04dbca26cb88](https://user-images.githubusercontent.com/8095975/175097853-e58c4be2-304d-4967-b2e7-1122914515b2.png)

After:
![cypress_006](https://user-images.githubusercontent.com/8095975/175097958-bd2bbbbc-b24a-4544-8ed8-a973d58690e3.png)


### PR Tasks


- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
